### PR TITLE
first design draft

### PR DESCRIPTION
**Pull request by Grayfoox**

### CODE REVIEW

File: design.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# design
```
Comment (bezruc) [inline design.md:1]: Please make some outline for what do you think is most important. At start, we will probably just make do with functioning planets/ships/moves, then we will implement from that point.

File: design.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+     - but focus steps can have "obstacles" that players needs to get to progress, stimulating trade(eg, slowing them)  
+   - after finishing focus step, everyone in game is informed about said group progress
+   - can range from casus beli, resource gain, making some choices cheaper (cheaper war declaration, diplomacy etc.) or just lore bits to motivate roleplay that still moves said group forward 
+   - can unlock/open/progress goals for group
+   - allows for snowball/catchup/burn-it-all options for every player/group
+   - (much later) opens posibility for small focus tree for every player for truly deep and complex roleplay/politics
+ - development/building
+   - now - simple "economy/politics/ship building" with numbered upgrades that gets exponecionally expensive
+ - ?technology
+   - (later versions) make a base default research tree
```
Comment (bezruc) [inline design.md:79]: will be in further stages. Probably in second or third.

File: design.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+   - using infuence points to suppre/promote any party on star. (always cheap to "defend" your stars)
+   - allows "overtake" over forgotten/abbadoned planets, allows for simple-yet-deep resource management on who and where pressure politically even in time of peace.
+ - capturing/fighting
+   - now - same as in neptune. higer number win, defender shoots first
+   - later - simple rock-paper-scissors ships to tie-in small bonus/lore/counterplays etc.
+   - even later - endless space "card strategy" system, every fleet would have a "strategy" picked from available imperium list of strategies, making small changes to fleet behaviour that are very slow and hard to change 
+     - things like "first strike - first round 30% more dmg , bunkerbuster - takes iniciative first againist def fleet, shield gating - if fleet would be destroyed last round, negate the damage once" etc. 
+     - make the simple one first ofc.
+     - tie in for roleplay/focus/tech 
+ - focus tree
```
Comment (bezruc) [inline design.md:67]: Waay in the future.

File: design.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+   - every political action should take at least one round to make other players able to react
+   - (later version) different groups can "add" or "oppose" to everyone else political actions (with penalty) 
+ - planetary politics (production of influence points)  
+   - stars generate influence points trough political power of population
+   - (similar to stellaris)
+   - (later) more population = more influence points created (population size can be from one type of building)
+   - star politics is percentage based, multiple groups can have political parties on any star and "siphon" influence points from someone else.
+   - using infuence points to suppre/promote any party on star. (always cheap to "defend" your stars)
+   - allows "overtake" over forgotten/abbadoned planets, allows for simple-yet-deep resource management on who and where pressure politically even in time of peace.
+ - capturing/fighting
```
Comment (bezruc) [inline design.md:60]: Seems we agree here.

File: design.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+ 
+ ### mechanics 
+ - trade 
+   - main resouces can be traded by players for whatever players agree on
+   - rare resouces should exists as more of a "optimalisation game" that stimulates activity - noncritical goods for imperium. Names and description can be mostly random and none of them should be "critical" to gameplay
+     - there should be more types of rare resources than half of the stars - no group should be able to control them all and is forced to trade with others
+     - every star generates and consume different rare resources
+     - if the star demand is not fulfilled, given star get small production penalty (max. 10%)
+     - comulative sum of all stars penalties should be allways displayed to motivate players to get rid of them
+ - politics
```
Comment (bezruc) [inline design.md:47]: later, we need working basics to politic over.

File: design.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+ - ?items (for later versions)
+   - group/player can obtain items that provide small boon/resources
+   - tradable, works as a small chaos agent to not make everything purely deterministic
+ - fleet
+   - military power of group
+   - used for taking up stars
+
+ 
+ ### mechanics 
+ - trade
```
Comment (bezruc) [inline design.md:40]: probably 2nd or 3rd to be implemented.

**Review by bezruc — COMMENTED** [review]
We need more of an incremental progression plan. If we plan some grand schemes, we will never get to finish atleast something.

File: design.md (hunk 1 above)
Comment (Grayfoox) [reply design.md:1]: we will split it into "roadmap" file to make it easier to orient

File: design.md (hunk 2 above)
Comment (Grayfoox) [reply design.md:79]: refer to roadmap

File: design.md (hunk 3 above)
Comment (Grayfoox) [reply design.md:67]: definitely in Q4/25

File: design.md (hunk 4 above)
Comment (Grayfoox) [reply design.md:60]: this is a key mechanic for politic focused players

File: design.md (hunk 6 above)
Comment (Grayfoox) [reply design.md:40]: trade is a core mechanic, if we get the core running other should follow simply.
now it can be a one way simple trade

**Comment by bezruc** [comment]
Oops

**Comment by bezruc** [comment]
Well, anyway. No clue wtf is going on.

**Pull request closed without merging**